### PR TITLE
fix: conflicts

### PR DESCRIPTION
**Pull request by EFF**
Sorry for the cryptic pr title. it simply fixes the stuff that were erased by bringing master to next

### CODE REVIEW

**Review by laurentlp — APPROVED** [review]
LGTM! If you see the ESLint test fail, just make sure to run `yarn eslint` locally and then fix each issue manually.

**Review by samuelmasse — DISMISSED** [review]
LGTM

**Review by samuelmasse — APPROVED** [review]

**Comment by samuelmasse** [comment]
Fixed the eslint problem

File: packages/studio-ui/src/web/reducers/ui.ts [hunk 1]
```diff
@@ -98,6 +98,12 @@ const reducer = handleActions(
         explorerOpen: value
       }
     },
+    [zoomIn]: (state, {}) => {
```
Comment (Raid55) [inline packages/studio-ui/src/web/reducers/ui.ts:101]: This is not imported, and cannot be imported either.

**Pull request merged**